build.gradle

```diff
@@ -1070,6 +1071,7 @@ project(':metadata') {
   archivesBaseName = "kafka-metadata"
 
   dependencies {
+    implementation project(':server-common')
```

Comment:
Not directly related to this PR. Do you know why we only include the storage module in releaseTarGz target, but not other server side modules?

Reply:
Good catch. It looks like these are already copied as part of `configurations.runtimeClasspath`. I raised https://github.com/apache/kafka/pull/10647 to fix that.